pa11y/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:14.5.0-alpine3.10
+
+# Installs latest Chromium (77) package.
+RUN apk add --no-cache \
+      chromium \
+      nss \
+      freetype \
+      freetype-dev \
+      harfbuzz \
+      ca-certificates \
+      ttf-freefont \
+      nodejs \
+      yarn
```

Comment:
We could *possibly* remove this and just use npm that comes with node - not sure if it changes how the package managing works but could save some time for the circle job to run if we only have to worry about one package manager.

Reply:
Removed installation of nodejs and yarn packages. Changed yarn command to npm install.